static/styles/zulip.css

```diff
@@ -4184,3 +4200,17 @@ li.show-more-private-messages a {
     }
 
 }
+
+.admin_emoji_table {
+    margin: 20px auto;
+}
+
+.admin_emoji_table th.actions,
+.admin_emoji_table th.image {width: 80px;}
+
+.emoji_image {width: 50px; display: block;}
+.emoji_image img {max-width: 100%;}
```

Comment:
I think stylistically our CSS always has a newline after the `{`, so probably should do that here too for consistency.